components/NavBar.vue

```diff
@@ -1,43 +1,27 @@
 <template>
   <div class="nav">
-    <nuxt-link to="/" class="brand">Bamboo Bicycle</nuxt-link>
-    <nav>
-      <nuxt-link to="/">Link 1</nuxt-link>&nbsp;|
-      <nuxt-link to="/create">Link 2</nuxt-link>
+    <!-- TODO: add real logo -->
+    <nuxt-link to="/" class="nav__logo"><img src="" />Bike Logo</nuxt-link>
+    <nav class="nav__link">
+      <nuxt-link to="/">Home</nuxt-link>
+      <nuxt-link to="/About">About us</nuxt-link>
+      <nuxt-link to="/Contact">Contact</nuxt-link>
     </nav>
   </div>
 </template>
 
 <style lang="scss">
-.brand {
-  font-family: 'Montserrat', sans-serif;
-  font-weight: 700;
-  font-size: 1.5em;
-  color: $color-green;
-  text-decoration: none;
-}
+@include font-roboto;
+
 .nav {
   display: flex;
   justify-content: space-between;
   align-items: center;
   height: 60px;
 }
-.nav .nav-item {
-  box-sizing: border-box;
-  margin: 0 rem(5);
-  color: $color-black;
-  text-decoration: none;
-}
+
 .nav .nav-item.router-link-exact-active {
```

Comment:
@mishschmid NuxtJS: Change the default NuxtLink classes and style the active and exact active classes (nuxt-link-active and nuxt-link-exact-active) https://nuxtjs.org/examples/routing/active-link-classes/
This does not work for Nuxt3, I had to use '.router-link-exact-active'.